Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,347 @@
+            var largestHistoricBalanceNames = GetLargestHistoricBalanceNames(people, largestHistoricBalance);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var largestHistoricBalanceNoComma = RemoveComma($"{largestHistoricBalance:C0}");
+
+            return $"{FixPlural(largestHistoricBalanceNames)} had the most money ever. {largestHistoricBalanceNoComma}.";
+        }
+
+        /// <summary>
+        /// Return name and loss of a person with a biggest loss (balance change negative).
+        /// </summary>
+        public static string FindPersonWithBiggestLoss(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var isBiggestLossValid = TryGetBiggestLoss(people, out var biggestLoss);
+
+            if (!isBiggestLossValid)
+            {
+                return NotAvailable;
+            }
+
+            var biggestLossNames = GetBiggestLossNames(people, biggestLoss);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            return $"{biggestLossNames} lost the most money. -{biggestLoss:C0}.";
+        }
+
+        /// <summary>
+        /// Return name and current money of the richest person.
+        /// </summary>
+        public static string FindRichestPerson(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var largestCurrentBalance = GetLargestCurrentBalance(people);
+            var largestCurrentBalancesNames = GetLargestCurrentNames(people, largestCurrentBalance);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var word1 = largestCurrentBalancesNames.ToString().Contains(", ") ? "are" : "is";
+            var word2 = largestCurrentBalancesNames.ToString().Contains(", ") ? "people" : "person";
+            var largestCurrentBalanceNoComma = RemoveComma($"{largestCurrentBalance:C0}");
+
+            return $"{FixPlural(largestCurrentBalancesNames)} {word1} the richest {word2}. {largestCurrentBalanceNoComma}.";
+        }
+
+        /// <summary>
+        /// Return name and current money of the most poor person.
+        /// </summary>
+        public static string FindMostPoorPerson(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var smallestCurrentBalance = GetSmallestCurrentBalance(people);
+            var smallestCurrentBalancesNames = GetSmallestCurrentBalanceNames(people, smallestCurrentBalance);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var minBalanceNoComma = RemoveComma($"{smallestCurrentBalance:C0}").Replace("(", "-").Replace(")", "");
+            var word = smallestCurrentBalancesNames.ToString().Contains(", ") ? "have" : "has";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)